### PR TITLE
Always show "online" for authenticated user profiles

### LINKED ISSUE #9570: Incorrect online status for authenticated account's profile
**Describe the bug**
Online status for the authenticated account is showing last online 9 minutes ago after staying logged in for a while when I myself have the tab opened 😅 

**To Reproduce**

1. Open my profile page

**Expected behavior**
For authenticated accounts, always show online. 

**Screenshots**
Image

### PR DESCRIPTION
**Pull request by abhimanyurajeesh**
## Proposed Changes

- Fixes #9570 
- Change 1: Modified UserStatusIndicator to always display "Online now" for the authenticated user viewing their own profile.

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Enhanced user status indicator to accurately reflect the online status of the authenticated user.
- **Style**
	- Minor formatting adjustment in the UserStatusIndicator component.

### CODE REVIEW

File: src/components/Users/UserListAndCard.tsx [hunk 1]
```diff
@@ -235,7 +235,7 @@ export const UserGrid = ({
 }: {
   users?: UserModel[] | UserAssignedModel[];
 }) => (
-  <div className="grid grid-cols-1 gap-4 @xl:grid-cols-3 @4xl:grid-cols-4 @6xl:grid-cols-5 lg:grid-cols-2"> 
+  <div className="grid grid-cols-1 gap-4 @xl:grid-cols-3 @4xl:grid-cols-4 @6xl:grid-cols-5 lg:grid-cols-2">
```
Comment (rithviknishad) [inline src/components/Users/UserListAndCard.tsx]: nice find

**Review by rithviknishad — APPROVED** [review]

**Comment by rithviknishad** [comment]
@abhimanyurajeesh 

Do add proper title for the PR.

Refer the following on getting started to contributing:

- https://github.com/ohcnetwork/care_fe?tab=readme-ov-file#contributing-to-care
- https://docs.ohc.network/docs/contributing

File: src/components/Users/UserListAndCard.tsx [hunk 2]
```diff
@@ -134,8 +134,11 @@ export const UserStatusIndicator = ({
   className?: string;
   addPadding?: boolean;
 }) => {
-  const cur_online = isUserOnline(user);
+  const authUser = useAuthUser();
+  const isAuthUser = user.id === authUser.id;
+  const cur_online = isUserOnline(user) || isAuthUser;
```
Comment (rithviknishad) [inline src/components/Users/UserListAndCard.tsx]: Although unrelated to the PR, while we are at it, can we ensure proper naming conventions being used?

ref: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#variable_names

```suggestion
  const isOnline = isUserOnline(user) || isAuthUser;
```

**Review by Jacobjeevan — APPROVED** [review]
LGTM, implement Rithvik's comment :)

File: src/components/Users/UserListAndCard.tsx (hunk 2 above)
Comment (abhimanyurajeesh) [reply src/components/Users/UserListAndCard.tsx]: Thank you for pointing out @rithviknishad, I have fixed it.

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**